webapp/components/emoji_picker/components/emoji_picker_preview.jsx

```diff
@@ -26,18 +27,36 @@ export default class EmojiPickerPreview extends React.Component {
                 // This is a system emoji which only has a list of aliases
                 name = emoji.aliases[0];
                 aliases = emoji.aliases;
-                previewImage = (<span className='sprite-preview'><img
-                    src='/static/images/img_trans.gif'
-                    className={'  emojisprite-preview emoji-' + emoji.filename + ' '}
-                                                                 /></span>);
+
+                if (this.props.category === 'recent') {
+                    previewImage = (
+                        <span className='sprite-preview'>
+                            <img
+                                className={'emoji-picker__preview-image'}
+                                src={EmojiStore.getEmojiImageUrl(this.props.emoji)}
```

Comment:
Shouldn't we still use the sprite-sheet CSS for recently used, non-custom emoji?